.github/workflows/pythonapp.yml

```diff
@@ -78,7 +78,7 @@ jobs:
     - name: python -m pip list
       run: python -m pip list
     - name: Install TVM from pypi if Ubuntu
-      if: ${{ matrix.os == 'ubuntu-18.04' }}
```

Comment:
We can use [startsWith](https://docs.github.com/en/actions/learn-github-actions/expressions#startswith) for string matching.

```yaml
if: ${{ startsWith(matrix.os, 'ubuntu') }}
```